### ISSUE TITLE
HelperText - lack of conveying status accurately

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly#7282]
**Describe the enhancement or change**
With v6 updates to helper text, additional updates still need to be made in example code:

- When there is a status helper text, there should be visually hidden text to provide context of that status like React does (see screenshot below)
  ![Image](https://github.com/user-attachments/assets/4753dddb-605a-4f18-8d2d-9e2c6e60a398)
- There are instances where a status helper text is being used, but no icon is rendered (examples and demos). Additionally, React no longer allows an icon to not be rendered if the helper text has a status, but Core still has a "Static" and "With icons" examples. Core should be updated to remove any example that conveys the icon being optional on anything that isn't a "default" helper text item.

**Any other information?**
